http-add-on/values.yaml

```diff
@@ -158,3 +158,69 @@ images:
 rbac:
   # -- Install aggregate roles for edit and view
   aggregateToDefaultRoles: false
+
+# -- [Security context] for all containers
+# @default -- [See below](#KEDA-is-secure-by-default)
+securityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+    - ALL
+  privileged: false
+  readOnlyRootFilesystem: true
+  runAsUser: 1000
+  runAsGroup: 1000
```

Comment:
We have faced with issues related with setting specific userIDs on OpenShift because it generates random userIDs for each pod. [Could you follow the approach that we have in KEDA core](https://github.com/kedacore/charts/blob/main/keda/values.yaml#L335-L392)?
It sets the minimum common parameters and suggests the other extra parameters with comments